02LessonTwo/homework/index.html

```diff
@@ -42,14 +42,19 @@ <h5 id="contact-info">Email: harrywada@gmail.com <br>
     <h3>Music</h3>
 
```

Comment:
Code looks good! Very organized and correctly used white space. 
You're missing an anchor element so this will stand at a 90% until you able to add some in (I suggest linking to the sites for the musicians or games).
Start getting used to commenting your code!